temporal/t.upgrade/t.upgrade.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+############################################################################
+#
+# MODULE:       t.upgrade
+# AUTHOR(S):    Martin Landa, Markus Neteler
+#
+# PURPOSE:      Upgrade of TGRASS DB
+# COPYRIGHT:    (C) 2019 by Martin Landa, and the GRASS Development Team
+#
+#  This program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#############################################################################
+
+# %module
+# % description: Upgrades the version of a space time dataset (TGRASS DB).
```

Comment:
What is TGRASS DB here is referred to as TGIS DB in the manual page. "version of a space time dataset" is likely confusing since the individual things are called datasets.

Reply:
I agree. We could be even more explicit and say: "Upgrades the version of the temporal database", given that t.upgrade is supposed to be ran only once per mapset and affect all stds, i.e., we do not run it on a per stds basis.